docs/users/plugins.md

```diff
@@ -128,4 +128,9 @@ formatted = mdformat.text(unformatted, extensions={"tables"})
     <td><code>deflist</code></td>
     <td>Adds support for <a href="https://pandoc.org/MANUAL.html#definition-lists">Pandoc-style</a> definition lists</td>
   </tr>
+  <tr>
+    <td><a href="https://github.com/KyleKing/mdformat-admon">mdformat-admon</a></td>
+    <td><code>admonitions</code></td>
```

Comment:
```suggestion
    <td><code>admonition</code></td>
```
I think this should match the string that is used here https://github.com/KyleKing/mdformat-admon/blob/2b55fb462fac5e4e6b1abfad1f3d27c75bd1209c/pyproject.toml#L36